jython-processors/trigonometric-transformation/processor.json

```diff
@@ -45,15 +43,50 @@ The way to read the measure on the resulting trigonometric circle is then the sa
             "name": "periodical_column",
             "label": "periodiclal column",
             "type": "COLUMN",
-            "description": "The column containing a periodical information (ex: day of year, day of month, day of week, month....)",
+            "description": "Column containing a periodical information",
             "mandatory": true
         },
+        {
+            "name": "process_as_time_dimension",
```

Comment:
Remove this parameter. We should handle this internally in the plugin code for valid date columns